ambry-api/src/main/java/com.github.ambry/notification/BlobType.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2017 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+package com.github.ambry.notification;
+
+/**
+ * Denotes the type of blob created. i.e. whether this is a simple blob, a metadata (composite) blob, or a data chunk
+ */
+public enum BlobType {
```

Comment:
There is a `BlobType` in MessageFormat api as well which is similar but is used to distinguish between blob content types - whether a blob contains data or metadata. Simple blobs as well as data chunk blobs are the same as far as MessageFormat is concerned.

This is similar, but I think here we are concerned more about what the blob id represents, which is notification system specific and for that does it make more sense to associate this type with the blob id? `BlobIdType` perhaps?